shuup/admin/modules/contacts/views/list.py

```diff
@@ -52,7 +53,10 @@ class ContactListView(PicotableListView):
         Column("groups", _("Groups"),
                filter_config=ChoicesFilter(ContactGroup.objects.all(), "groups"),
                display="get_groups_display"),
-        Column("shops", _("Shops"), filter_config=Select2Filter(Shop.objects.all()), display="get_shop_display")
+
+        # TODO: Fix these filters, Shop.objects.all() is wrong...
+        Column("shops", _("Shops"), filter_config=Select2Filter(Shop.objects.all()), display="get_shop_display"),
```

Comment:
Maybe we can't have this filters at all if we can't limit the shop. You also have always option to add text filter and allow option to filter by shop name. Pretty much the same thing.

Reply:
Yea, I think this will be fixed as a part of https://github.com/shuup/shuup/issues/1403

Reply:
So does this really filter the shops somehow. Surely we can't merge this if all the shops end up in the filter. The #1403 is not on roadmap either and might be something we need to wait first wave of admin updates before doing.

Reply:
This is the same way as it's been done before (as you can see, I just moved it) But I can look into this